ingest-mixed-generator/messages.py

```diff
@@ -174,3 +186,35 @@ def user_report_message_generator(idx: int, settings: Mapping[str, Any]) -> byte
 
 def default_message_generator(idx: int, settings: Mapping[str, Any]) -> bytes:
     return event_message_generator(idx, settings, event_type="default")
+
+
+def generate_event_messages(idx: int, settings: Mapping[str, Any], payload: bytes):
```

Comment:
Let's rename this function to make it super explicit that it's producing attachments, and that it's also an **additional** way to generate them.